docs/firmware/azure/Index.rst

```diff
@@ -9,6 +9,12 @@ It is compatible with the :ref:`nRF Asset Tracker for Azure <index_azure>`.
 
 The `firmware GitHub repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-firmware-azure>`_ provides an example for using the application as an out-of-tree development copy, which optionally enables the SUPL client.
 
+.. note::
+
+   The current firmware-over-the-air (FOTA) update mechanism implementated in nRF Asset Tracker for Azure is based on `this tutorial <https://docs.microsoft.com/en-us/azure/iot-hub/tutorial-firmware-update>`_.
```

Comment:
```suggestion
   The current firmware-over-the-air (FOTA) update mechanism implemented in nRF Asset Tracker for Azure is based on the `Microsoft tutorial on implementing a device firmware update process<https://docs.microsoft.com/en-us/azure/iot-hub/tutorial-firmware-update>`_.
```

Reply:
First change is fixed, second change conflicts with @peknis suggestion. But I guess I'll take that.